recipes/rmlui/3.3/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RmluiConan(ConanFile):
+    name = "rmlui"
+    description = "RmlUi - The HTML/CSS User Interface Library Evolved"
+    homepage = "https://github.com/mikke89/RmlUi"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    topics = ("conan", "css", "gui", "html", "lua", "rmlui")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "build_lua_bindings": [True, False],
+        "disable_rtti_and_exceptions": [True, False],
+        "enable_precompiled_headers": [True, False],
```

Comment:
Does this option change anything about the resulting binary?
If this option is only internal to the build process of `rmlui`, then this option might be removed (and kept to its default value).

Reply:
No, it doesn't. I delved a bit into the documentation and the only thing it needs is CMake support, starting from a certain version. I implemented a tiny bit of logic that will toggle this behind-the-scenes. Thanks for the suggestion!

Reply:
Thanks for checking it!